nautobot/extras/test_jobs/singleton.py

```diff
@@ -0,0 +1,16 @@
+import time
+
+from nautobot.core.celery import register_jobs
+from nautobot.extras.jobs import Job
+
+
+class TestSingletonJob(Job):
+    class Meta:
+        name = "Example Singleton Job"
+        is_singleton = True
+
+    def run(self):
+        time.sleep(10)
```

Comment:
Can we change this to a `1` to save a bit more test run time in `test_job_ignore_singleton`?

Reply:
Works, done!